### DISCUSSION TITLE
Bump the version to something > 3.0.0 ?

### DISCUSSION THREAD

**Discussion by AskBojesen** [dotnet-bluetooth-le/dotnet-bluetooth-le#767] [General]
Hi @janusw : Shouldn't you bump the version in master after the 3.0.0 release?
(In order for us to use the new packages from master)

**Comment by janusw** [comment]
Well,  I usually only bump the version whenever I make a release that is uploaded to nuget.org. The packages built via GHA on the current master branch have the version `3.0.0-34-g0790051` (this version code is simply generated via `git describe`). If you build a package yourself from the sources, it will probably still get a version 3.0.0, unless you explicit set some version. You can see how this is done in GHA here: https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/blob/07900518cb30eff6e71b8fc4ee7f6541a67723c2/.github/workflows/dotnet.yml#L63

**Reply by janusw** [reply]
Oh, and yes, I'll probably tag some version 3.1.0-beta soon (but probably not before next week). Until then, I hope the GHA-built packages can act as a makeshift solution for everybody who quickly needs to try the latest developments on master.

**Comment by AskBojesen** [comment]
Ok, thanks for the explanation
